src/iocmme.cpp

```diff
@@ -304,6 +306,16 @@ void CmmeInput::ReadEvents(pugi::xml_node eventsNode)
                 pugi::xml_node barlineNode = eventNode.select_node("./Barline").node();
                 CreateBarline(barlineNode);
             }
+            else if (eventNode.select_node("./Ellipsis")) {
```

Comment:
You don't need an xpath here. Just `evenNode.child("Ellispis")` is enough, which give you the node straight way
```suggestion
            else if (eventNode.child("Ellipsis")) {
```